### ISSUE TITLE
Chapters without sections get empty running title on right-hand pages

### ISSUE THREAD

**Issue by alexis-michaud** [langsci/langscibook#151]
Right-hand pages' headers contain the first-level section heading, thus: 
![image](https://github.com/langsci/langscibook/assets/6072524/8316eecf-af7d-4b5c-94e4-88609980942b)

Chapters without sections get empty running title on right-hand pages. That tends to happen for short chapters, such as conclusion chapters. 
An example is the conclusion of [LangSci 109](https://langsci-press.org/catalog/book/109), on page 445: 
![image](https://github.com/langsci/langscibook/assets/6072524/570243b4-0d77-4eaa-96dc-e6611a99e5b1)
(It only affects that one page, because the "Conclusion" chapter has just four pages.)

Another example is the conclusion of [LangSci 391](https://langsci-press.org/catalog/book/391). 
![image](https://github.com/langsci/langscibook/assets/6072524/a631f9cd-5db2-497f-a266-8f28d64b29cf)
Here we have a hefty conclusion, and as many as eight pages are affected.

The fix would consist in treating headers of chapters lacking sections in a way that resembles how headings are handled for front/back matter (acknowledgments, references...): having the same heading on left & right pages. (A difference is that short chapters as the "Conclusion" in the two examples above carry a number, and hence the heading needs to contain the chapter number alongside the chapter title.)

**Comment by kopeckyf** [comment]
Thank you for the feature request. In back- and front matter sections like prefaces, the list references, or acknowledgements, the section title is usually only printed in the outer verso heading. So the same effect that you mention happens there and there is not really an asymmetry in the LSP layout. I'll have a look around whether there is a technically sound way of achieving what you have mind. It could certainly help in some situations, but I'm not sure whether it's easy to implement.

**Comment by alexis-michaud** [comment]
Hey @kopeckyf, great to read your answer!

> In back- and front matter sections like prefaces, the list references, or acknowledgements, the section title is usually only printed in the outer verso heading. So the same effect that you mention happens there and there is not really an asymmetry in the LSP layout. 

Ooooh wow, how strange! I had not realized that the list of references had that asymmetry in published books. 
So it is! E.g. [Nadine Grimm's book](https://langsci-press.org/catalog/book/298):
![image](https://github.com/langsci/langscibook/assets/6072524/c606ab1c-4d84-44b0-a912-73e0a06d7020)

_That's_ strange! and not too good-looking, I feel?

> I'll have a look around whether there is a technically sound way of achieving what you have mind. It could certainly help in some situations, but I'm not sure whether it's easy to implement.

Hey, my book has it, so the code that Luise Dorenbusch produced for my book could help others too? :smirk: 
Yes my 2017 book has the "References" section title on outer verso & inner recto headings
![image](https://github.com/langsci/langscibook/assets/6072524/b2ff633f-0249-43b2-9a20-b478c6f5fdfa)

The code is [here](https://github.com/langsci/109) of course. As I remember, Luise & I agreed that it looked like a bug, and she fixed it. I can't vouch for it that it's technically sound, though :sweat_smile: but I would expect that it's OK, as (to the extent that I can judge) her work was really professional.

**Comment by alexis-michaud** [comment]
Looking at the code of [LangSci\109](https://github.com/langsci/109/blob/509a4f99614caea59a73461ff7b228efb3972727/indexed/abbreviations.tex#L4), here's a quick fix that was used for the added 'Abbreviations & conventions' section:
```
% Add missing right-page heading
\rohead{Abbreviations and conventions}

```
(line 5)

**Comment by kopeckyf** [comment]
No, unfortunately manual adjustments like you propose are not suitable for mass deployment. The risk of error is too high.

Whether or not there is even an issue will need to be discussed internally with the press directors (there's also a possible line of argument which says everything is fine since the recto header should be for subsections only, which aren't there in these cases). If there's a decision on how things should be, I'll be looking into ways to implement it. 

I'll keep you updated.

**Comment by kopeckyf** [comment]
Hi @alexis-michaud, and sorry about the long time we needed to get back to you. 

In https://github.com/langsci/langscibook/commit/81ec60afaf83c1ef4e4c965a31213f8b973e6228 I have optimised the headers for References sections in LSP monographs (see below). It will probably take some time before this change is applied to new titles, but you can update your langscibook.cls if you don't want to wait. Basically the change is achieved with `\markdouble` from `scrbook`:

https://github.com/langsci/langscibook/blob/81ec60afaf83c1ef4e4c965a31213f8b973e6228/langscibook.cls#L1585-L1587

I will be moving this issue to our [latex-skeletons](https://github.com/langsci/latex-skeletons) repository, because the issue for Prefaces etc needs to be sorted out there.

![grafik](https://github.com/user-attachments/assets/2d7273e8-21c1-4ade-880b-752fec49ebdb)
